header-check/Makefile

```diff
@@ -0,0 +1,13 @@
+PROJECT = header-check
+MELANGE_CONTEXTDIR ?= /tmp/melange-context/$(PROJECT)
+MELANGE_INSTALL_PATH = $(MELANGE_CONTEXTDIR)/usr/bin
+
+.PHONY: build melange-install
+
+build:
+	chmod +x header-check
+
+melange-install: build
+	echo $@
```

Comment:
(`make` by default is going to print the command it runs, so if this results in output like:
```
$ make melange-install
echo melange-install
melange-install
```

and thats just kind of silly. If you want the `echo` there, then i'm ok with that, but put a `@` in front of it and maybe `@echo running $@`, so you get:

```
$ make melange-install
running melange-install
```

Reply:
I copied that straight from ldd-check.

Reply:
Deleted the echo.  Perhaps ldd-check should do the same?